test/DefaultUserTests.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from "chai";
+import "mocha";
+import { IConfigCatClient, User } from "../src";
+import { FakeConfigCatKernel, FakeConfigFetcherWithRules } from "./ConfigCatClientTests";
+import * as configcatClient from "../src/index";
+import { assert } from "chai";
+
+
+describe("DefaultUser", () => {
+
+  it("Default user set works", async () => {
+    const redEyeColorUser = { identifier: 'redIdentifier', custom: { 'eyeColor': 'red' } };
+    const blueEyeColorUser = { identifier: 'blueIdentifier', custom: { 'eyeColor': 'blue' } };
+
+    let configCatKernel: FakeConfigCatKernel = { configFetcher: new FakeConfigFetcherWithRules(), sdkType: "common", sdkVersion: "1.0.0" };
+    var client: IConfigCatClient = configcatClient.createClientWithAutoPoll("APIKEY", configCatKernel, {logger: configcatClient.createConsoleLogger(configcatClient.LogLevel.Debug)});
+
+    // Default
```

Comment:
Please be a bit more specific about what these tests are about

Reply:
What about now?